tests/unit/test_subscription_request.py

```diff
@@ -29,7 +29,7 @@ def test_build_request_success(geom_geojson):
             "start_time": "2021-03-01T00:00:00Z",
             "end_time": "2023-11-01T00:00:00Z",
             "rrule": "FREQ=MONTHLY;BYMONTH=3,4,5,6,7,8,9,10",
-            "item_types": ["PSScene"],
+            "item_types": "PSScene",
```

Comment:
this file seems to not be updated from my previous review, as the brackets are still missing